Client/src/Pages/Settings/index.jsx

```diff
@@ -37,6 +38,7 @@ const Settings = ({ isAdmin }) => {
     ttl: checkTTL ? (checkTTL / SECONDS_PER_DAY).toString() : 0,
   });
   const [errors, setErrors] = useState({});
+  const [isOpen, setIsOpen] = useState({delMntrs: false, delSts: false});  
```

Comment:
This doesn't make semantic sense, `isOpen` suggests a `boolean` value, so having it be an object doesn't make sense.

It's also a compound object that represents two states that is controlling two different modals.

I think it would be much clearer and simpler to separate these into two states:

`const [isDeleteMonitorsOpen, setIsDeleteMonitorsOpen] = useState(false);`
`const [isDeleteStatsOpen, setIsDeleteStatsOpen] = useState(false);`

Our goal should always be simplicity, maintainability, and readability :+1:

Reply:
Alex, if we has updates on both states at the same time (setting one as false, while setting the other as true), maybe we could go with the object state approach, and come up with a better name for the state. Updating two states can lead to performance issues, do you agree?

Reply:
updated in : https://github.com/bluewave-labs/bluewave-uptime/pull/939/commits/21caca1ffbc9f624d9bc9a68283495f4278083af